recipes/python-highcharts/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.2.0" %}
+{% set name = "python-highcharts" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: aeacac36fafd9c1c37df88cead05a344
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - jinja2
+    - future
+
+test:
+  imports:
+    - highcharts
+    - highcharts.highcharts
+    - highcharts.highmaps
+    - highcharts.highstock
+
+about:
+  home: https://github.com/kyper-data/python-highcharts
+  license: MIT
```

Comment:
Raised the issue on the git repo. Thanks @jakirkham 

https://github.com/kyper-data/python-highcharts/issues/14